stable/nginx-ingress/README.md

```diff
@@ -60,6 +60,8 @@ Parameter | Description | Default
 `controller.replicaCount` | desired number of controller pods | `1`
 `controller.resources` | controller pod resource requests & limits | `{}`
 `controller.service.annotations` | annotations for controller service | `{}`
+`controller.publishService.enabled` | enables the additional `--publish-service` controller setting | `false`
```

Comment:
Let's make this clearer. Maybe a shortened version of the flag's [help text](https://github.com/kubernetes/ingress/blob/5e6624404f14346f1e3e4887004e5cacb2f14bd5/core/pkg/ingress/controller/launch.go#L50)?